Core/Base/AgentModule.py

```diff
@@ -381,31 +389,44 @@ def am_go(self):
     self.log.notice(" Average execution/polling time: %.2f%%" % elapsedPollingRate)
     if cycleResult['OK']:
       self.log.notice(" Cycle was successful")
+      if self.activityMonitoring:
+        self.activityMonitoringReporter.addRecord({
+            'timestamp': datetime.datetime.utcnow(),
```

Comment:
I am wondering whether I did not tell you something wrong before. Please wait for @zmathe comment as whether the timestamp should be epoch or utc

Reply:
We store epoch in ES, but the Monitoring system accept epoch(int) time and utc datetime as well. If the timestamp is datetime, it will be converted to epoch. For performance reason I prefer epoch, because it will be easier to serialize and send through the network ...

Reply:
Okay so I will go with epoch.